cmd/sks_kubeconfig.go

```diff
@@ -33,7 +35,47 @@ Example usage:
 Note: if no TTL value is specified, the API applies a default value as a
 safety measure. Please look up the API documentation for more information.
 
+## Using exo CLI as Kubernetes credential plugin
+
+If you wish to avoid leaving sensitive credentials on your system, you can use
+exo CLI as a Kubernetes client-go credential plugin[2] to generate and return
+a kubeconfig dynamically when invoked by kubectl without storing it on disk.
+
+To achieve this configuration, edit your kubeconfig file so that the
+"users" section relating to your cluster ("my-sks-cluster" in the following
+example) looks like:
+
+    apiVersion: v1
+    kind: Config
+    clusters:
+    - name: my-sks-cluster
+      cluster:
+        certificate-authority-data: **BASE64-ENCODED CLUSTER CERTIFICATE**
+        server: https://153fcc53-1197-46ae-a8e0-ccf6d09efcb0.sks-ch-gva-2.exo.io:443
+    users:
+    - name: exo@my-sks-cluster
+      user:
+        # The "exec" section replaces "client-certificate-data"/"client-key-data"
+        exec:
+          apiVersion: "client.authentication.k8s.io/v1beta1"
+          command: "exo"
+          args: ["sks", "kubeconfig", "my-sks-cluster", "--exec-credential", "user"]
```

Comment:
I think we should also add the zone here.

Reply:
Good point, added 👍